bitcoin/src/blockdata/witness.rs

```diff
@@ -133,18 +170,17 @@ impl Witness {
     /// Creates [`Witness`] object from an array of byte-arrays
     pub fn from_vec(vec: Vec<Vec<u8>>) -> Self {
```

Comment:
Oh, why are we even requiring `Vec`? `pub fn from_slice<B: AsRef<[u8]>>(slice: &[B])` would be more useful. (Unrelated ofc)